src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class Tests {
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
+	private static ArrayList<CreditCardVendor> vendors = reader.returnVendorArray();
+	private static CreditCardValidator validator = new CreditCardValidator();
+	private static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	private static CreditCardVendor vendorMasterCard = vendors.get(1);
+	
```

Comment:
## Idea of refactor
If we separate Validator features (length check , mask check , general vendor-independent check)
we can preform tests more precisely per vendor.

Vendor could be Enum having vendor-based checks (length and mask related)  encapsulated inside.
To still have flexibility of redefining masks/length in config file
the entry in config can match enum name.

We can then define Parametrized test 
going over range of positive (and separate negative) cases for given vendor.

Then vendor-related part can be in separate Unit Test class, non-vendor in yet another.
Combination of validations (defined in Validator) could be then in ValidatorTest.
All to separately focus on single feature

Then we can define such test methods:
- shouldSucceedOnCorrectNumberLenght(String number, CreditCardVendor vendor)
- shouldReturnFalseForIncorrectNumberLenght(String number, CreditCardVendor vendor)
- shouldSucceedOnIINCheck(String number, CreditCardVendor vendor)
- shouldReturnFalseOnIINCheck(String number, CreditCardVendor vendor)

Then content of vendor-based parametrized tests (mocked vendors with differen masks as params) could be 
```
   vendor.validateIINMaskCorrectness(number);
```